coprocessors/flake.nix

```diff
@@ -0,0 +1,85 @@
+{
+  description = "Coprocessor NixOS configurations for 3636's 2024 FRC robot.";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-23.11";
+    nixos-rk3588 = {
+      url = "github:ryan4yin/nixos-rk3588";
+      inputs.nixpkgs.follows = "nixpkgs";
+    };
+    rgb-2024 = {
+      url = "github:FRC3636/rgb-2024";
+      inputs.nixpkgs.follows = "nixpkgs";
+    };
+  };
+
+  outputs = { self, nixpkgs, nixos-rk3588, nixos-hardware, rgb-2024, }:
+    let inherit (nixpkgs) lib;
+    in rec {
+      colmena = {
+        meta = {
+          nixpkgs = import nixpkgs {
+            system = "x86_64-linux";
+            overlays = [ overlays.default ];
+          };
+          specialArgs = { rk3588 = nixos-rk3588.inputs; };
+        };
+
+        copepod = {
```

Comment:
it'd be nice if we could document somewhere what the hostnames actually refer to. e.g. i think the tangerine one is the mini pc

Reply:
+1